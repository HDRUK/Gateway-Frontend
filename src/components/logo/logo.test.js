import React from "react";
import { create } from "react-test-renderer";
import Logo from "./logo";

describe("<Paragraph> ", () => {
    let renderedComponent;

    beforeEach(() => {
        renderedComponent = create(<Logo />);
    });

    it("should test the NewsTile Snapshot", () => {
        expect(renderedComponent).toMatchSnapshot();
    });

    it("should check the correct style elements are displayed", () => {
        const img = renderedComponent.root.findAllByType("img");

        expect(img).toHaveLength(1);
    });
});
