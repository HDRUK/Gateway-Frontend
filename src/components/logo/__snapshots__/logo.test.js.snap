// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph>  should test the NewsTile Snapshot 1`] = `
<img
  src="hdruk_logo_white.png"
/>
`;
